src/Symfony/Component/Form/Button.php

```diff
@@ -309,11 +317,11 @@ public function isRequired()
      */
     public function isDisabled()
     {
-        if (null === $this->parent || !$this->parent->isDisabled()) {
-            return $this->config->getDisabled();
+        if ($this->parent && $this->parent->isDisabled()) {
```

Comment:
`null === $this->parent`?

Reply:
Not needed.

Reply:
Actually I just reversed the logic here for readability